style.css

```diff
@@ -0,0 +1,34 @@
+img {
+  display: block;
+  margin: 0 auto;
+  height: auto;
+  max-width: 100%;
+  padding: 10px;
+}
+
+h4 {
+  font-weight: lighter;
+}
+
+h1 h3 h4 {
```

Comment:
I would suggest you give these headings a class so as to make your output looks nice as seen in the real project output
The

element is the generic container for flow content. It has no effect on the content or layout until styled in some way using CSS (e.g. styling is directly applied to it, or some kind of layout model like [Flexbox] is applied to its parent element.
You can access this link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div for more explanation on div tag

Reply:
- [ ] Instead of writing in this way you can write like -: "Kindly use class for headings so you can use class for styling purposes instead of adding all headings for styling. Using class will make your code short and precise. Also using class follows the best CSS practices.
_Good job in proving a link for reference._

Reply:
Thank you .I just made the changes